### PR TITLE
Chore: Cleanup warnings

### PR DESCRIPTION
**Pull request by ConnorBP**
Added some null type checking as well as removed unused imports to remove as many warnings as possible. Have not removed the deprication warning for `clazz.newInstance()` though. https://github.com/jensjeflensje/minecraft_imagery/blob/16658fcaf4bfcc25c357e24fa3f4f266593d6b65/src/main/java/dev/jensderuiter/minecraft_imagery/compatibility/PatchInitializer.java#L44

This one would need your input.

### CODE REVIEW

**Comment by ConnorBP** [comment]
@jensjeflensje is this good to merge?

**Comment by jensjeflensje** [comment]
I'll take a closer look and merge later, but I think it looks good.

The only thing I'm not quite sure about is the build.cmd script. I understand why you put it in, but it only supports windows, only works when your jdk is in that specific path, and I don't think anyone would really use it. I use IntellIJs builtin maven tooling and I think it's better to just have people use that, anything else, rather than putting it inside the repo.

File: src/main/java/dev/jensderuiter/minecraft_imagery/image/ImageCapture.java [hunk 1]
```diff
@@ -338,7 +339,7 @@ private RayTraceResult rayTraceEntitiesFromList(Location start, Vector direction
             }
         }
 
-        return nearestHitEntity == null ? null : new RayTraceResult(nearestHitResult.getHitPosition(), nearestHitEntity, nearestHitResult.getHitBlockFace());
+        return (nearestHitEntity == null || nearestHitResult == null) ? null : new RayTraceResult(nearestHitResult.getHitPosition(), nearestHitEntity, nearestHitResult.getHitBlockFace());
```
Comment (jensjeflensje) [inline src/main/java/dev/jensderuiter/minecraft_imagery/image/ImageCapture.java:342]: Why would this be needed?

File: build.cmd
Comment (jensjeflensje) [inline build.cmd:1]: Could we remove this file?

**Review by jensjeflensje — CHANGES_REQUESTED** [review]

File: src/main/java/dev/jensderuiter/minecraft_imagery/image/ImageCapture.java (hunk 1 above)
Comment (ConnorBP) [reply src/main/java/dev/jensderuiter/minecraft_imagery/image/ImageCapture.java:342]: Suppresses linter warnings, and has no real impact on the code

File: build.cmd
Comment (ConnorBP) [reply build.cmd:1]: Yup, I'll remove that

**Comment by ConnorBP** [comment]
@jensjeflensje sorry for the delay, the script is removed

**Comment by jensjeflensje** [comment]
Great! Thanks for the PR :)
It'll be included in the next release

**Pull request merged**